.github/workflows/test.yml

```diff
@@ -281,15 +281,15 @@ jobs:
         key: ubuntu22.04-gpuocelot-19626fc00b6ee321638c3111074269c69050e091
         restore-keys: |
           ubuntu22.04-gpuocelot-19626fc00b6ee321638c3111074269c69050e091
-    - if: ${{ steps.cache-build.outputs.cache-hit != 'true' }}
```

Comment:
why did this change?

Reply:
For some reason the cached version of gpuocelot does not pass tests even with no changes from master (see a few commits above). This change is not meant to be merged; I will revert it once you approve this patch. Can you try clearing the gh actions cache? (Wozeparrot suggested this cause/line of action)

Reply:
I don't understand, there's 20 other PRs on this repo passing.

Reply:
https://github.com/tinygrad/tinygrad/pull/1141/files/ffbdfcdb9c0433a08175f1c4ea5ea3728cc8b4b3

I don't understand either.